### ISSUE TITLE
0 (Zero) values are not supported

### ISSUE THREAD

**Issue by JMuratha** [vaadin/vaadin-combo-box#287]
if you have an items array like this `[{value: 0, label: 'pls support me'}, {value: 1, label: 'I am supported'}]` the `vaadin-combo-box` is bugging around when you want to select the item with the value 0.

I know there shouldn't be Entries in the Database with the ID 0, but it's an existing one so I can't change it.

Simple fix for this:

in the vaadin-combo-box-behavior.html on line 483 replace this function with this Code:

```
    _indexOfValue: function (value, items) {
          items = items || this.items;
          if (items) {
            for (var i = 0; i < items.length; i++) {
              if (String(this._getItemValue(items[i])) === String(value)) {
                return i;
              }
           }
         }

       return -1;
    },
```

this will also support null values (if you want that)

I can't do a pull request because i already have another change on my fork.

**Comment by tehapo** [comment]
This is similar issue to #202

As described in https://github.com/PolymerElements/iron-list/issues/171, the `iron-list` element does not work with zero or empty string items until version 1.2.0 upwards.

**Comment by JMuratha** [comment]
Lukily i have the version 1.3.5.

But i think this should still be fixed that 0 values are not supported, because its an easy fix.

**Comment by tehapo** [comment]
Investigated this a bit further and seems like your patch still does not fix the case of using just number values as `items` instead of objects.

But you're right, this definitely should be fixed and we will look into this. Thanks for reporting!

**Comment by Saulis** [comment]
Fixed by 1cc8d72
